auth0/Guardian.Android
# feat: rich consents

By submitting a PR to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](https://github.com/auth0/.github/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

Adds fetch rich consent record capability to guardian sdk.


### References


### Testing

1. Run the sample app
2. Enroll to MFA
3. Initiate a CIBA authentication request for the enrolled user
4. Verify the consent details are displayed



### Checklist
